api/admin/model/quicksight.py

```diff
@@ -1,18 +1,21 @@
 from typing import List
+from uuid import UUID
 
 from pydantic import Field, validator
 
 from core.util.flask_util import CustomBaseModel, str_comma_list_validator
 
 
 class QuicksightGenerateUrlRequest(CustomBaseModel):
-    library_ids: List[int] = Field(
+    library_uuids: List[UUID] = Field(
         description="The list of libraries to include in the dataset, an empty list is equivalent to all the libraries the user is allowed to access."
     )
 
-    @validator("library_ids", pre=True)
-    def parse_library_ids(cls, value):
-        return str_comma_list_validator(value)
+    @validator("library_uuids", pre=True)
+    def parse_library_uuids(cls, value) -> List[UUID]:
+        uuid_list = str_comma_list_validator(value)
+        # verify that all strings in the list are UUIDs
+        return list(map(UUID, uuid_list))
```

Comment:
This works as is, but you don't actually need to create `UUID` objects in this validator. Since its marked as `pre=True` it runs before Pydantic's usual data conversion. So if we return a `List[str]` Pydantic will handle turning it into a `List[UUID]`. 

Something like the validation function before this change.
```python
    @validator("library_uuids", pre=True)
    def parse_library_uuids(cls, value) -> List[str]:
        return str_comma_list_validator(value)
```

Reply:
Ah - okay.  I'm remove the unnecessary conversion and merge.